CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Improve twitter share _community pr!_ ([#6690](https://github.com/lbryio/lbry-desktop/pull/6690))
 - Update lighthouse search api _community pr!_ ([#6731](https://github.com/lbryio/lbry-desktop/pull/6731))
 - Update sockety api _community pr!_ ([#6747](https://github.com/lbryio/lbry-desktop/pull/6747))
+- Use resolve for OG metadata instead of chainquery _community pr!_ ([#6787](https://github.com/lbryio/lbry-desktop/pull/6787))
```

Comment:
Not sure where the error is, but Embed OG's aren't showing up.
Example embed: https://odysee.com/$/embed/eevblog-1412-argon-ion-laser-teardown!/1fe325659bf8d791f72e02bb614a942fa4b571bf?r=5yqd2A3cQ59DhkNRYTJq3BApzdcr7KSn

Reply:
The embed url doesn't use `:` or `#` to separate the id.

Reply:
Adding this `.replace('\/', '#');` fixes the issue. Not sure if this will work on all cases.

Reply:
As long as embed url is always `$/embed/name/id`  ^^